packages/order-utils/src/store/order_filled_cancelled_lazy_store.ts

```diff
@@ -58,9 +60,10 @@ export class OrderFilledCancelledLazyStore implements AbstractOrderFilledCancell
      * @param orderHash OrderHash from order of interest
      * @return Whether the order has been cancelled
      */
-    public async getIsCancelledAsync(orderHash: string): Promise<boolean> {
+    public async getIsCancelledAsync(signedOrder: SignedOrder): Promise<boolean> {
```

Comment:
Is this a new convention? That no function should accept a hash but should recompute it on it's own?

Reply:
No, it is not. The reason we need it is because `isOrderCancelledAsync` in `OrderFilledCancelledFetcher` needs not only the OrderHash, but also the order's `makerAddress` and `signerAddress` to check both the `cancelled` and `cancelledOrdersUpTo` mappings, and so we just pass in the order.